src/libs/actions/Session/index.ts

```diff
@@ -482,27 +482,40 @@ function signUpUser() {
 function signInAfterTransitionFromOldDot(transitionURL: string) {
     const [route, queryParams] = transitionURL.split('?');
 
-    const {email, authToken, encryptedAuthToken, accountID, autoGeneratedLogin, autoGeneratedPassword, clearOnyxOnStart} = Object.fromEntries(
+    const {email, authToken, encryptedAuthToken, accountID, autoGeneratedLogin, autoGeneratedPassword, clearOnyxOnStart, shouldReturnToOldDotAfterBooking} = Object.fromEntries(
```

Comment:
Hmmm in case of your question 🤔 I'm not sure if we can achieve that easily. There are some necessary screens on NewDot that should be always in the navigation structure so It might result in more problems than benefits 🤔